tools/pipelines/server-historian.yml

```diff
@@ -80,7 +80,7 @@ pr:
     - tools/pipelines/templates/include-vars.yml
     - tools/pipelines/templates/include-install-pnpm.yml
     - tools/pipelines/templates/include-use-node-version.yml
-    - tools/pipelines/templates/upload-dev-manifest.yml
+
```

Comment:
I still maintain we should keep these here for the 3 server pipelines, because we could technically break them by making changes to this file, and if they don't trigger during PRs we wouldn't know. Of note, we removed them from the `pr:` section but not the `trigger:` section, so this is problematic anyway.

Reply:
`include-publish-npm-package.yml` template is not included in the `pr:` section. Is it still expected to add `upload-dev-manifest.yml` under the `pr:` section?

Reply:
Oh, interesting... I can see why we would have skipped the publish templates for `pr:`, because they are completely skipped conditionally in the public project so they end up not affecting the resulting YML... and then the PR build wouldn't really tell us anything even if we broke the pipelines by making changes to those files. Ok, I think it's fine to remove them here then, and in that case we should also remove it from build-client's `pr:` section.